app/views/common_parameters/_parent_parameter.html.erb

```diff
@@ -7,7 +7,7 @@
           <%= content_tag(:input, '', :class => "form-control", :disabled => true,  :value => _('Global'), :type=> :text ) %>
         </td>
         <td class="col-md-2">
-          <%= text_field_tag("parent_parameter_#{i}_key", '', :class => "form-control", :placeholder => parameter.name) %>
+          <%= text_field_tag("parent_parameter_#{i}_key", '', :class => "form-control", :value => parameter.name) %>
```

Comment:
I won't change this for now so @unorthodoxgeek can also be included in the discussion next week.